MSAL/test/app/mac/MSALAcquireTokenViewController.m

```diff
@@ -56,11 +56,10 @@ @interface MSALAcquireTokenViewController ()
 @property (weak) IBOutlet NSSegmentedControl *webViewSegment;
 @property (weak) IBOutlet NSSegmentedControl *validateAuthoritySegment;
 @property (weak) IBOutlet NSStackView *acquireTokenView;
-@property (weak) IBOutlet WKWebView *webView;
 @property (weak) IBOutlet NSPopUpButton *userPopup;
 @property (weak) IBOutlet NSSegmentedControl *authSchemeSegment;
 
-
+@property (nonatomic) WKWebView *webView;
```

Comment:
Any reason to make as ```nonatomic```

Reply:
Good catch. I guess there's no reason to make this nonatomic, as we want to make sure any operation on the webView object is thread-safe. it's atomic by default, and we want thread operation to be serialized on this object.